templates/main.html

```diff
@@ -12,9 +12,13 @@
       <div class="row">
       {% for staff in staff %}
         <div class="col-md-4 col-sm-6">
-          <a href="/profile/{{ staff.username }}">
-          {% include "user_profile.html" %}
-          </a>
+            <a href="/profile/{{ staff.username }}">
```

Comment:
We can use `url_for` here and for all `a href`s too. `<a href="{{ url_for('profile', uname=staff.username) }}">`.